server/src/main/java/org/candlepin/resource/CrlResource.java

```diff
@@ -84,7 +84,7 @@ public Response getCurrentCrl(@Context Principal principal) throws CRLException
             // Create an empty CRL if we didn't have anything to write
             if (!crlFile.exists() || crlFile.length() < 1) {
                 pkiUtility.writePemEncoded(
-                    pkiUtility.createX509CRL(new LinkedList<>(), BigInteger.ZERO),
+                    pkiUtility.createX509CRL(new LinkedList<>(), BigInteger.ONE),
```

Comment:
Did we have an off-by-one issue here?

Reply:
No, there was another spot in the code where we were calling `createX509` but with BigInteger.ONE.  This just makes it consistent across the codebase.